packages/errors/src/codes.ts

```diff
@@ -16,8 +16,8 @@ export const SOLANA_ERROR__MULTIPLE_ACCOUNTS_NOT_FOUND = 9 as const;
 export const SOLANA_ERROR__FAILED_TO_DECODE_ACCOUNT = 10 as const;
 export const SOLANA_ERROR__EXPECTED_DECODED_ACCOUNT = 11 as const;
 export const SOLANA_ERROR__NOT_ALL_ACCOUNTS_DECODED = 12 as const;
-export const SOLANA_ERROR__INCORRECT_BASE58_ADDRESS_LENGTH = 13 as const;
-export const SOLANA_ERROR__INCORRECT_BASE58_ADDRESS_BYTE_LENGTH = 14 as const;
+export const SOLANA_ERROR__ADDRESS_BYTE_LENGTH_OUT_OF_RANGE = 13 as const;
+export const SOLANA_ERROR__ADDRESS_STRING_LENGTH_OUT_OF_RANGE = 14 as const;
```

Comment:
Massaged these names while I was here. tl;dr I'm converting `blockhash.ts` and it's exactly the same as `address.ts`.